app/models/project.rb

```diff
@@ -7,4 +7,6 @@ class Project < ApplicationRecord
 
   has_many :project_users
   has_many :users, through: :project_users
-end
\ No newline at end of file
+
```

Comment:
remove extra line

Reply:
This line gets added by default as my editors setting. Rubocop objects for not having last blank line.